clouddriver-web/clouddriver-web.gradle

```diff
@@ -63,9 +63,14 @@ jar {
 }
 
 startScripts {
-  defaultJvmOpts = applicationDefaultJvmArgs + ["-Dspring.config.location=/opt/spinnaker/config/"]
+   defaultJvmOpts = applicationDefaultJvmArgs + ["-Dspring.config.location=/opt/spinnaker/config/"]
   doLast {
     unixScript.text = unixScript.text.replace('DEFAULT_JVM_OPTS=', '''\
+        java_version=$(java -version 2>&1|head -1)
+        if [[ "$java_version" != *"1.8.0"* ]]; then
```

Comment:
I think we should split the java_version on ".", and pull out the second field, since it IS the major version now, and run with that.